powerquery-docs/connectors/windsor.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Power Query Windsor connector
+description: Provides basic information and prerequisites for the Windsor connector, supported authentication types, outlines limitations and issues.
+author: lanceris
+ms.author: miescobar
+ms.topic: conceptual
+ms.date: 06/19/2024
+---
+
+# Windsor Power Query custom connector
+
+>[!Note]
+>The following connector article is provided by Windsor, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the https://windsor.ai/ website and use the support channels there.
+
+## Summary
+| Item | Description                                                                       |
+| ---- |-----------------------------------------------------------------------------------|
+| Release State | Preview                                                                     |
+| Products | Power BI (Datasets)<br/>Power BI (Dataflows)<br/> Power Apps (Dataflows) |
+| Authentication Types Supported | OAuth                                  |
+| Function reference docs | &mdash; |
+
+
+## Prerequisites
+>Before using the connector, you need to create a [Windsor](https://onboard.windsor.ai/) account and save at least one query.
+
+Follow these steps to create and save a query:
+
+1. [Sign in to the Windsor.ai account.](https://onboard.windsor.ai/login) via Email + Password, Microsoft account or Google account.
+
+2. Select a data source and sign in to your platform account to create a connection between your data source and Windsor.
+
+3. Click "Next (Data Preview)" bottom left or "2. Preview and Destination" on top.
+
+4. Configure your query using controls on the right side and click on "Get Data" button, if present, to preview data.
+    - Select Data Source to fetch data from. Note: by default "all" will be selected, which will get data from all data source you have connected.
+    - Define a date range - use a flexible date range, for example “Last 7 days", and always get data for the previous seven days.
+    - Choose fields. [See our full list of supported fields](https://windsor.ai/data-field/all/)
+    - Pick accounts to fetch data from.
+
+5. When you are satisfied with the result, click on Save Query button, give your query a recognizable name and click "Save".
+    
+    :::image type="content" source="media/windsor/windsor-save-query.png" alt-text="Save Query at Windsor Onboard.":::
+
+    The queries you save here are available to you in Power BI Desktop once you sign in to the Windsor connector.
+
+
+## Capabilities supported
+* Import 
+* Data import from multiple sources
```

Comment:
You can delete this line. You can either have Import mode and/or DIrectQuery. "Data import from multiple sources" would fall in the "Import" category